src/ts/i18n/fallbackLanguage.ts

```diff
@@ -7,7 +7,6 @@ const fallbackLanguage : Language = {
 	"author": "Computernewb",
 
 	"stringKeys": {
-		"kGeneric_CollabVM": "CollabVM",
```

Comment:
Why are we removing this? This is a used translation node (even to this changeset).